### DISCUSSION TITLE
Is the GitHub release and the NPM release different?

### DISCUSSION THREAD

**Discussion by Paraxdev** [mantinedev/mantine#107] [Q&A] [answered]
Since GitHub shows that the 1.1.1 release is the latest and NPM says that 1.2.1 is the latest I'm wondering if this is just a error or an NPM only fix?

**Comment by rtivital** [comment, marked as answer]
Please see this comment – https://github.com/mantinedev/mantine/discussions/87#discussioncomment-772421

**Reply by Paraxdev** [reply]
Ohh ok thanks ^^ i was a bit confused
